### DISCUSSION TITLE
Welcome to vscode-antlr4 Discussions!

### DISCUSSION THREAD

**Discussion by mike-lischke** [mike-lischke/vscode-antlr4#175] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by startibartfast** [comment]
Hello,
my name is Carlo Ieva and I'm using your tool while developing a DSL for an insurance pricing engine. 
First of all, thanks for your great job, your extension is really helpful and I am wondering whether you could help me in fixing a small issue. When I generate RRD diagram I can't save it neither the single one nor the 'all rules' one. When I click on the 'Save to HTML' or 'Save to SVG' button nothing happens. 
Any help would be greatly appreciated.
Thanks in advance,
Carlo

**Reply by mike-lischke** [reply]
Looks like a bug, but has not been reported as such. Please open an issue for that, so I can plan for it when working on the extension.

**Reply by startibartfast** [reply]
Thanks, I'll do it
